src/app/components/EmbedConsentBanner/index.tsx

```diff
@@ -3,12 +3,15 @@ import { SocialEmbedProviders } from '../../models/types/global';
 
 import ConsentBanner from './ConsentBanner';
 
-const CONSENT_BANNER_PROVIDERS: SocialEmbedProviders[] = ['youtube'];
+export const CONSENT_BANNER_PROVIDERS: Extract<
+  SocialEmbedProviders,
+  'youtube' | 'tiktok'
+>[] = ['youtube', 'tiktok'];
 
-interface ConsentBannerProps {
-  provider: SocialEmbedProviders;
+type ConsentBannerProps = {
+  provider: typeof CONSENT_BANNER_PROVIDERS[number];
```

Comment:
I don't really understand what this type means 😅 I'm expecting `number` to be in scope somewhere?

Reply:
I may change this to be a bit more readable, although _slightly_ less type face. A fair compromise as its hard to reason about it.